modules/system/console/MixWatch.php

```diff
@@ -60,6 +60,12 @@ public function handle(): int
             sprintf('Watching package "%s" for changes', $name)
         );
         $this->mixJsPath = $relativeMixJsPath;
+
+        // Create Windows signal handler
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN' && function_exists('sapi_windows_set_ctrl_handler')) {
+            sapi_windows_set_ctrl_handler([$this, 'handleWindowsSignal'], true);
+        }
```

Comment:
The conflicting signal handling code merged with @bennothommo's commit https://github.com/wintercms/winter/commit/452bb4222351b1a6d3cc0288a30804ebeb37234d